clients/admin-ui/src/features/config-wizard/AuthenticateOktaForm.tsx

```diff
@@ -32,18 +32,44 @@ import ScannerLoading from "./ScannerLoading";
 
 const initialValues = {
   orgUrl: "",
-  token: "",
+  clientId: "",
+  privateKey: "",
+  scopes: "okta.apps.read",
 };
 
 type FormValues = typeof initialValues;
 
 const ValidationSchema = Yup.object().shape({
-  orgUrl: Yup.string().required().trim().url().label("URL"),
-  token: Yup.string()
+  orgUrl: Yup.string().required().trim().url().label("Organization URL"),
+  clientId: Yup.string()
     .required()
     .trim()
     .matches(/^[^\s]+$/, "Cannot contain spaces")
-    .label("Token"),
+    .label("Client ID"),
+  privateKey: Yup.string()
+    .required()
+    .trim()
+    .test(
+      "is-valid-json",
+      "Private key must be valid JSON. Paste the JWK downloaded from Okta.",
+      (value) => {
+        if (!value) {
+          return true;
+        }
+        try {
+          JSON.parse(value);
+          return true;
+        } catch {
+          return false;
+        }
+      },
+    )
+    .label("Private Key"),
+  scopes: Yup.string()
+    .required()
+    .trim()
+    .label("Scopes")
+    .default("okta.apps.read"),
```

Comment:
Since it does the split / etc below - should this also sanity check that the text field is either one string or a list of string with commas?